segment_ytm.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+)
+
+type ytm struct {
+	props   *properties
+	env     environmentInfo
+	service ytmStatusService
+}
+
+const (
+	// YTMDARemoteControlAPIURL is the YTMDA Remote Control API URL property.
+	YTMDARemoteControlAPIURL Property = "ytmda_remote_control_api_url"
+)
+
+func (y *ytm) string() string {
+	// Hit the Remote Control API to find out what song is playing (or paused).
+	// https://github.com/ytmdesktop/ytmdesktop/wiki/Remote-Control-API
+	status, err := y.service.Get()
+	if err != nil {
+		return ""
+	}
+
+	if status.state == stopped {
+		return y.props.getString(StoppedIcon, "\uF04D ")
+	}
+
+	icon := ""
+	separator := y.props.getString(TrackSeparator, " - ")
+	if status.state == paused {
+		icon = y.props.getString(PausedIcon, "\uF8E3 ")
+	} else {
```

Comment:
I would switch state, if else is to be avoided in my book ;-)

Reply:
Bit rusty with Go. Updated.